_63C/digitalwallet.md

```diff
@@ -0,0 +1,14 @@
+---
+question: C9
+title: Can a digital identity wallet be an IdP?
+---
+
+Digital identity wallets do not normally fulfill the requirements to be considered an IdP by the Digital Identity Guidelines. There has been interest in recent years in using applications to store cryptographically protected material on a user-carried device, a practice sometimes referred to as a digital identity wallet.  In their most common applications, digital identity wallets act as queryable personal data stores, collecting attributes from a variety of sources and presenting these attributes directly to relying parties.
+
+In most distributed scenarios that digital identity wallets are applied to, the RP's trust in the attributes tends to come from the ultimate sources of the attributes, not from the wallet itself. This is in contrast to the IdP model presented in the Guidelines, where the RP's trust in the attributes stems from its trust in the IdP asserting those attributes in a federation transaction. From the perspective of the Guidelines, the IdP is the source of the attributes. The IAL and AAL for a given federation transaction are applied to the entire account being presented, with any attributes asserted as being a part of that account.
+
+Furthermore, while the IdP model in the Guidelines does allow for presentation of attributes through an identity API in addition to being present in the assertion itself (see [Q-C8](#q-c8) for details), even in these cases, the identity API is assumed to be under the control of the IdP and inherits its trust from the IdP. Advanced use cases involving independent attribute providers, attribute metadata (including provenance), and distributed attribute models are outside the scope of the Guidelines.
```

Comment:
```suggestion
Furthermore, while the IdP model in the Guidelines does allow for presentation of attributes through an identity API in addition to being present in the assertion itself (see [Q-C8](#q-c8) for details), even in these cases, the identity API is assumed to be under the control of the IdP and inherits its trust from the IdP. Use cases involving independent attribute providers, attribute metadata (including provenance), and distributed attribute models are outside the scope of the Guidelines.
```
The issue isn't that we don't consider "advanced" use cases, but the difficulty of scaling trust to one IdP per subscriber.

Reply:
The one IDP per person isn't the focus of this sentence, but I'm fine with pulling back on the adjective.